packer/ecs/root/etc/systemd/system/ecs-agent.service

```diff
@@ -27,7 +27,7 @@ ExecStart=/usr/bin/docker run \
     --publish=127.0.0.1:51679:51679 \
     --env-file=/etc/ecs/ecs.config \
     --env=ECS_CLUSTER=${SERVER_GROUP} \
-    amazon/amazon-ecs-agent:v1.14.1
+    amazon/amazon-ecs-agent:latest
```

Comment:
Dont know if this is a great idea. It happened to me twice that they introduce a critical bug in the latest version and screwed up all my cluster. Personally I would prefer to have that stuck to a fixed version